pkg/oc/bootstrap/docker/up.go

```diff
@@ -304,6 +304,15 @@ func (c *ClusterUpConfig) Complete(f *osclientcmd.Factory, cmd *cobra.Command, o
 	c.dockerClient = client
 	taskPrinter.Success()
 
+	// Ensure that the OpenShift Docker image is available.
+	// If not present, pull it.
+	// We do this here because the image is used in the next step if running Red Hat docker.
+	taskPrinter.StartTask(fmt.Sprintf("Checking if image %s is available", c.openshiftImage()))
```

Comment:
This doesn't appear to be setting value sin the `Options` struct.  Move it to the `Run`?

Reply:
see my comment "We do this here because the image is used in the next step..." where next step is next line of code ;-)

literally this code block fails on "CanUseNsenterMounter" tries to run the image.

```
	// Check whether the Docker host has the right binaries to use Kubernetes' nsenter mounter
	// If not, use a shared volume to mount volumes on OpenShift
	if isRedHatDocker, err := c.DockerHelper().IsRedHat(); err == nil && isRedHatDocker {
		taskPrinter.StartTask("Checking type of volume mount")
		c.UseNsenterMount, err = c.HostHelper().CanUseNsenterMounter()
		if err != nil {
			return taskPrinter.ToError(err)
		}
		taskPrinter.Success()
	}
```

i don't love where the code is right now, i tried to put it next to startKubelet until i saw the above.